src/components/1-Importance/IntroMap.vue

```diff
@@ -2726,50 +2726,50 @@
                     <text
                       id="philly-dot"
                       class="city-dot"
-                      transform="translate(304.1 506.1) scale(11.1 11.1)"
+                      transform="translate(304.1 506.1) scale(10 10)"
                     >•</text>
```

Comment:
Curious what the issue was with safari, and if you found any logic to why it's different there? I've struggled with predicting safari weirdness. Sometimes it seems to be less flexible with how units and positioning are declared.

Reply:
It was odd - the map was displaying on desktop more like it does on mobile, so the symbol sizes were off. I didn't dig into why - more just tried to adjust things so that it will (hopefully) look better.